cypress/integration/Modal.spec.ts

```diff
@@ -141,7 +152,7 @@ describe('Modal', () => {
         context('when the overlay is clicked', () => {
           beforeEach(() => {
             cy.findByLabelText('Delete Item')
-              .pipe(h.modal.getOverlay)
+              .parent()
               .click('top');
```

Comment:
This should be `cy.get('body').click('top')` instead. I don't think we should specify _which_ element is clicked, just that we're clicking outside the modal.